### ISSUE TITLE
review

### ISSUE THREAD

**Issue by alexisrc1** [mihirparadkar/world-development#11]
The report is very clear, and the different steps of the reasoning well explained: describing the features, how your work could be used by governments, how the data is split, 50% training and 50% testing (even though Pr Udell mentioned in class that usually it is more 80% for the estimation sample and 20% for the validation sample).
But what I particularly liked when reading your report is the use of the Generalized Low Ranked models package that allowed you to use different regularizers and compare the outcome. Also, even if you did mention why you used the Huber loss for the loss function (because of its capacity to handle outliers), it would have been nice to compare the results given by other loss functions.
I have to admit that I was very impressed with the math you put in your projects (the use of the Laplacian Matrix, the explanation of the proximal gradient method…), how you chose the rank (hyper parameter determination)....
Overall, I found this project very interesting and I learnt a lot about how to fill a dataset with missing values on it. 
Congrats!
